### PR TITLE
Remove loess fit

### PR DESCRIPTION
**Pull request by saifulbkhan**
This set of patches not only removes the deprecated "Loess fit" alignment feature, but also fixes the issues in visual appearance of elements in Alignment dialog when viewed on low resolution screens.

### CODE REVIEW

File: src/gui/mzroll/background_peaks_update.cpp [hunk 1]
```diff
@@ -485,54 +414,6 @@ void BackgroundPeakUpdate::align() {
                         aligner.doAlignment(groups);
                         mainwindow->sampleRtWidget->setDegreeMap(aligner.sampleDegree);
                         mainwindow->sampleRtWidget->setCoefficientMap(aligner.sampleCoefficient);
-                } else if (alignAlgo == 2) {
-                        mainwindow->getAnalytics()->hitEvent("Alignment", "LoessFit");
-                        aligner.preProcessing(groups, mavenParameters->alignWrtExpectedRt);
```
Comment (shubhra-agrawal) [inline src/gui/mzroll/background_peaks_update.cpp]: `Aligner::preProcessing` is not being used anywhere. It was written to communicate data to the python executable from what I understand.

File: src/gui/mzroll/background_peaks_update.cpp [hunk 2]
```diff
@@ -485,54 +414,6 @@ void BackgroundPeakUpdate::align() {
...
-                                else{
-                                        errorMessage=errorMessage+"<br>"+"Incomplete data, re-run alignment";
-                                }
-                                qDebug()<<errorMessage;
-                                Q_EMIT alignmentError(errorMessage);
-                                return;
-                        }
-
-                        if(!parentObj.isEmpty()){
-                                aligner.updateRts(parentObj);
```
Comment (shubhra-agrawal) [inline src/gui/mzroll/background_peaks_update.cpp]: `Aligner::updateRts` is not being used anywhere and can be removed

File: src/gui/mzroll/background_peaks_update.cpp (hunk 1 above)
Comment (saifulbkhan) [reply src/gui/mzroll/background_peaks_update.cpp]: Removed.

File: src/gui/mzroll/background_peaks_update.cpp (hunk 2 above)
Comment (saifulbkhan) [reply src/gui/mzroll/background_peaks_update.cpp]: Deleted these as well.

**Pull request merged**